source/includes/fact-olson-tz-behavior.rst

```diff
@@ -0,0 +1,58 @@
+When using an Olson Timezone Identifier in the ``<timezone>``
+field, MongoDB applies the :abbr:`DST (Daylight Savings Time)` offset
+if applicable for the specified timezone.
+
+For example, consider a ``sales`` collection with the following document:
+
+.. code-block:: javascript
+
+   {
+      "_id" : 1,
+      "item" : "abc",
+      "price" : 20,
+      "quantity" : 5,
+      "date" : ISODate("2017-05-20T10:24:51.303Z")
+   }
+
+The following aggregation illustrates how MongoDB handles the DST
+offset for the Olson Timezone Identifier:
+
+.. cssclass:: copyable-code
+.. code-block:: javascript
+
+   db.sales.aggregate([
```

Comment:
I was initially confused because I had expected to see $dateToParts/$dateFromParts on the rendered page for this.
 I think explicitly stating that you are using :expression:`$hour`  may help this.